Tasks/DockerComposeV0/dockercomposebuild.ts

```diff
@@ -60,6 +61,7 @@ function addOtherTags(connection: DockerComposeConnection, imageName: string): a
 export function run(connection: DockerComposeConnection): any {
     var command = connection.createComposeCommand();
     command.arg("build");
+    command.line(dockerCommandUtils.getCommandArguments(tl.getInput("arguments", false)));
```

Comment:
I worry about null reference with this code. Can you please make tl.getInput('',..) in 1 line and assign it to variable. Based on non-nullity of it get the command arguments in the next line and based on the non-nullity of it, use command.line(…)

Reply:
Done. Checked for null references